hostutil/host_remote.go

```diff
@@ -108,5 +108,8 @@ func SetupCommand(host *host.Host) string {
 	if host.Distro.SetupAsSudo {
 		cmd += " --setup_as_sudo"
 	}
+
+	cmd += fmt.Sprintf("--working_directory=%s", host.Distro.WorkDir)
```

Comment:
Need a space in the beginning of the string.

Reply:
done